utils/forms.ts

```diff
@@ -1,34 +1,23 @@
-import CHILD_FORMS from 'data/googleForms/childForms';
-import ADULT_FORMS from 'data/googleForms/adultForms';
+import ADULT_FORM_NAMES from 'data/formFilterNames/adultFormNames';
+import CHILD_FORM_NAMES from 'data/formFilterNames/childFormNames';
 
 import { getPermissionFlag, UserPermissions } from 'utils/user';
 
-type Form = typeof ADULT_FORMS[number];
+const sortForms = (formA: string, formB: string) =>
+  formA.toLocaleLowerCase() < formB.toLowerCase() ? -1 : 1;
 
-interface IForm extends Form {
-  id?: string;
-}
-
-interface NormalisedForm {
-  text: string;
-  value: string;
-}
-
-const sortForms = (formA: NormalisedForm, formB: NormalisedForm) =>
-  formA.text.toLowerCase() < formB.text.toLowerCase() ? -1 : 1;
-
-const normaliseForms = (forms: IForm[]) =>
-  forms.map(({ id, text }) => ({ text, value: id ?? text }));
+const adminArray: Array<string> = [
+  ...ADULT_FORM_NAMES,
+  ...CHILD_FORM_NAMES,
+].sort(sortForms);
 
 export const getFormsByUserPermission = (
   user: UserPermissions
-): Array<NormalisedForm> => {
+): Array<string> => {
   const permission = getPermissionFlag(user);
   return permission === 'C'
-    ? normaliseForms(CHILD_FORMS).sort(sortForms)
+    ? CHILD_FORM_NAMES.sort(sortForms)
     : permission === 'A'
-    ? normaliseForms(ADULT_FORMS).sort(sortForms)
-    : [...normaliseForms(ADULT_FORMS), ...normaliseForms(CHILD_FORMS)].sort(
-        sortForms
-      );
+    ? ADULT_FORM_NAMES.sort(sortForms)
+    : adminArray.filter((u, i) => i === adminArray.indexOf(u));
```

Comment:
What is doing this line? I am a bit confused 🤔 
`adminArray.filter((u, i) => i === adminArray.indexOf(u))`

Reply:
Both lists have names which are the same such as case audit, if your an admin, you get duplicates within the filter list, this line just removes the duplication if you are an admin